docs/conf.py

```diff
@@ -70,9 +70,11 @@
 
 intersphinx_mapping = {
     'python': ('https://docs.python.org/3', None),
-    'packaging': ('https://packaging.python.org/en/latest/', None),
+    'packaging': ('https://packaging.python.org/', None),
 }
 
+# https://github.com/pypa/packaging.python.org/issues/995
```

Comment:
This comment seems misplaced. Are you confusing the documentation site dedicated to the Python Packaging Guides vs the library called `packaging`?
```suggestion
```